bluewin.ch

```diff
@@ -1,41 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
-  <emailProvider id="bluewin.ch">
+  <emailProvider id="swisscom.ch">
     <domain>bluewin.ch</domain>
-    <!-- See also bluemail.ch -->
-    <displayName>bluewin.ch</displayName>
-    <displayShortName>bluewin.ch</displayShortName>
+    <domain>bluemail.ch</domain>
```

Comment:
You need to add bluewin.ch as domain, if you want this config to apply to foo@bluewin.ch as well as foo@bluemail.ch :
```suggestion
    <domain>bluewin.ch</domain>
    <domain>bluemail.ch</domain>
```

Reply:
<domain>bluewin.ch</domain> is on line 4 so just not visible in the PR view but it is already there

Reply:
correct, sorry